.github/workflows/build.yml

```diff
@@ -22,11 +22,20 @@ on:
 name: build
 
 jobs:
-  build:
+  php-built-in-server:
     uses: yiisoft/actions/.github/workflows/codeception.yml@master
     with:
       extensions: fileinfo, intl
       os: >-
-        ['ubuntu-latest']
+        ['ubuntu-latest', 'windows-latest']
+      php: >-
+        ['8.1', '8.2', '8.3']
+  yii-serve:
```

Comment:
Only this one should stay since we need to use our own tool instead of a workaround.

Reply:
done.